src/test/JerryProtocolHandler.test.ts

```diff
@@ -361,7 +371,7 @@ suite('JerryProtocolHandler', () => {
 
     test('aborts when unhandled message sent', () => {
       delegate.onError.resetHistory();
-      const array = Uint8Array.from([SP.SERVER.JERRY_DEBUGGER_CONFIGURATION, 200, 4, 1, 5]);
+      const array = createConfiguration([SP.SERVER.JERRY_DEBUGGER_CONFIGURATION, 1, 200, 4]);
```

Comment:
This  is the first array where the message type was set correctly because it will leads to an error with `0` here. :)
Please, write the first element of the config blocks to this `SP.SERVER.JERRY_DEBUGGER_CONFIGURATION` constant everywhere, even though the `onConfiguration`  function does not check the first element.